fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/AbstractService.java

```diff
@@ -238,8 +238,6 @@ private Statement createDefaultAccessToStatement(final String authSubject) {
                         createResource(currentResourcePath));
     }
 
-    protected String txId(final Transaction tx) {
-        return tx == null ? null : tx.getId();
-    }
-
-}
+    protected String txtIdIfUncommittedOrNull(final Transaction tx) {
```

Comment:
should this be `txIdIfUncommittedOrNull` rather than txtId?

Reply:
This should be removed I believe.

Reply:
I thought I saw it being referenced a few places, but if the extra `getResource(Transaction tx...)` is added then this should be unneeded